jsettlers.mapcreator/src/jsettlers/mapcreator/main/DataTester.java

```diff
@@ -49,21 +52,48 @@
 	private final ErrorList errorList;
 	private ArrayList<Error> errors = new ArrayList<Error>();
 
+	/**
+	 * Thread for testing
+	 */
+	private Thread thread;
+
+	/**
+	 * Running flag
+	 */
+	private boolean running = true;
+
 	public DataTester(MapData data, TestResultReceiver receiver) {
 		this.data = data;
 		this.receiver = receiver;
 		errorList = new ErrorList();
 	}
 
-	public void start() {
-		new Thread(this, "data tester").start();
+	public synchronized void start() {
+		running = true;
+		this.thread = new Thread(this, "data tester");
+		thread.start();
+	}
+
+	/**
+	 * Release all resources
+	 */
+	public void dispose() {
+		running = false;
+		synchronized (this) {
+			notifyAll();
+		}
+		try {
+			thread.join(100);
```

Comment:
I didn't see your commend, github didn't automatically refresh...
I'll fix the Datatester, and see if I can improve it a little bit with the new sidebar.
I'll extends my swing exception handler in the Map Editor. It can be later moved to use also for the game. Now I'm only working on the Map Editor, I don't know the Code enough good.
(And the Map Editor don't have to run on Android, makes it for the beginning a lot easyer...)

Logging is then may the next task...